finance_dl/amazon.py

```diff
@@ -226,11 +251,11 @@ def get_source():
                 f.write('\ufeff' + page_source)
             logger.info('  Wrote %s', invoice_path)
 
-    def run(self):
+    def run(self, *args, **kwargs):
```

Comment:
Instead of making these parameters of run, how about making regular and digital options of __init__, with regular defaulting to True and digital defaulting based on domain?